pkg/config/agentmanagement.go

```diff
@@ -181,6 +184,12 @@ func (am *AgentManagementConfig) SleepTime() (time.Duration, error) {
 	return time.ParseDuration(am.PollingInterval)
 }
 
+// jitterTime returns a random duration in the range [0, am.PollingInterval).
+func (am *AgentManagementConfig) JitterTime() time.Duration {
+	maxJitterDuration, _ := am.SleepTime()
+	return time.Duration(rand.Int63n(int64(maxJitterDuration)))
```

Comment:
LGTM overall but let's allow some level of configuration here? (for example a percentage of the sleep time with some acceptable ranges)